### ISSUE TITLE
weave not compatible with latest k8s

### ISSUE THREAD

**Issue by matrohon** [networkservicemesh/networkservicemesh#1642]
## Current Behavior

Following the quick-start guide leads to the following error:

```
$ kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
serviceaccount/weave-net configured
clusterrole.rbac.authorization.k8s.io/weave-net configured
clusterrolebinding.rbac.authorization.k8s.io/weave-net configured
role.rbac.authorization.k8s.io/weave-net configured
rolebinding.rbac.authorization.k8s.io/weave-net configured
unable to recognize no matches for kind "DaemonSet" in version "extensions/v1beta1"
```

This error has been fixed upstream in the weave repo by https://github.com/weaveworks/weave/pull/3660

To fix it for nsm deployments, I had to manually install latest version of weave with:

```
kubectl apply -f "https://raw.githubusercontent.com/weaveworks/weave/master/prog/weave-kube/weave-daemonset-k8s-1.9.yaml"
```

**Comment by alex-yust** [comment]
@denis-tingajkin  looks like #1438 will fix the issue, won't it?

**Comment by denis-tingajkin** [comment]
probably yes

**Comment by alex-yust** [comment]
@matrohon, #1438 is merged and I checked that there is no error reported on weave installation

Hope that they will release https://github.com/weaveworks/weave/pull/3660 by the time we switch to k8s 1.16

**Comment by haiodo** [comment]
Closing since with current version of Kubernetes working fine. Fix in #1438

**Comment by bboreham** [comment]
cloud.weave.works should be fixed now.
